test2.js

```diff
@@ -0,0 +1,19 @@
+var foo = '' ;
+
+if(foo  === "bar") {
+  console.log( 'hey' )
+}
+
+
+var foo88 = '' ;
+
+if(foo88  === "b88") {
+  console.log( 'he88' );
```

Comment:
Codacy has a fix for the issue: [Replace `·'he88'·` with `"he88"`](https://release.k8s.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=5796)
```suggestion
  console.log("he88");
```